src/tests/ftest/server/replay.py

```diff
@@ -19,40 +23,39 @@ class ReplayTests(TestWithServers):
     :avocado: recursive
     """
 
-    def test_restart(self):
-        """Verify data access after engine restart w/ WAL replay + w/ check pointing.
-
-        Tests un-synchronized WAL & VOS
-
-        Steps:
-            0) Start 2 DAOS servers with 1 engines on each server (setup)
-            1) Create a single pool and container
-            2) Run ior w/ DFS to populate the container with data
-            3) After ior has completed, shutdown every engine cleanly (dmg system stop)
-            4) Remove VOS file manually/temporarily (umount tmpfs; remount tmpfs)
-            5) Restart each engine (dmg system start)
-            6) Verify the previously written data matches with an ior read
+    def create_container(self, details=None):
+        """Create a pool and container.
 
-        :avocado: tags=all,pr
-        :avocado: tags=hw,medium
-        :avocado: tags=server,replay
-        :avocado: tags=ReplayTests,test_restart
+        Returns:
+            TestContainer: the created container with a reference to the created pool
         """
-        processes = self.params.get('ppn', '/run/ior_write/*', 1)
-        self.log_step('Creating a pool and container')
+        self.log_step(join(' - ', 'Creating a pool (dmg pool create)', details))
         pool = add_pool(self)
+        self.log_step(join(' - ', 'Creating a container (daos container create)', details))
         container = self.get_container(pool)
+        return container
```

Comment:
Nit - Do we need a local reference?
```suggestion
        return self.get_container(pool)
```

Reply:
Done